app/view/main/Map.js

```diff
@@ -177,6 +199,138 @@ Ext.define('CpsiMapview.view.main.Map', {
             });
         }
 
+        if (me.enablePermalink) {
+            if (window.location.hash !== '') {
+                // try to restore center, zoom-level and rotation from the URL
+                me.applyState(me.readPermalink(window.location.hash));
+            }
+
+            me.registerPermalinkEvents();
+        }
+
         Ext.GlobalEvents.fireEvent('cmv-mapready', me);
+    },
+
+    /**
+     * Registers the events to have a permalink synchronization.
+     *
+     * @private
+     */
+    registerPermalinkEvents: function () {
+        var me = this;
+
+        // update permalink when visible map state changes
+        me.olMap.on('moveend', me.updatePermalink, me);
+
+        // restore the view state when navigating through the history, see
+        // https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onpopstate
+        window.addEventListener('popstate', function(event) {
```

Comment:
According to the link above this is supported for IE10 and higher, and also in every sane browser. This [bugreport](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/3740423/) mentions issues in IE 11 / Edge …

I think it would be good if we culd test this in an IE, and also if @geographika may confirm this works for him. I want this to be in, so he can easily test this.

Reply:
@marcjansen - tested in Edge and IE11, both work fine. 
I tried IE10 with the emulator option in IE11 but the page doesn't load (with or without permalinks). 
For the prototype it is fine to target IE11 and Edge only. So LGTM.

Reply:
Thanks a ton for this info, @geographika.

Reply:
Great, thanks for testing @geographika!